web/src/utils.js

```diff
@@ -60,6 +60,30 @@ function getDandisetContact(dandiset) {
   return null;
 }
 
+const basicDandisetSortingOptions = [
```

Comment:
The default sort of the API is on `meta.dandiset.identifier`. Is there a reason that's not an option presented in the GUI?

Reply:
Since this is basically a duplicate of the public dandisets page, I don't have a good reason, but that could be a useful addition.